shared/src/main/java/org/geysermc/floodgate/shared/util/ModCommandUtil.java

```diff
@@ -73,8 +71,9 @@ protected Collection<?> getOnlinePlayers() {
 
     @Override
     public boolean hasPermission(Object source, String permission) {
-        return Permissions.check((SharedSuggestionProvider) source,
-                permission, MinecraftServerHolder.get().getOperatorUserPermissionLevel());
+        return FloodgateMod.getCommandManager().hasPermission(
```

Comment:
if this is the only case where the command manager needs to be get'd statically, i would put a static `hasPermission` on FloodgateMod instead